ide-support/revliburl.livecodescript

```diff
@@ -3333,7 +3333,7 @@ on libUrlFtpUpload pData,pUrl,pMessage
   
   put false into lvJumpOut
   put ulStripUrl(pUrl) into newUrl
-  if char 1 to 3 of newUrl <> "ftp" then
+  if char 1 to 3 of newUrl <> "ftp" and char 1 to 4 of newUrl <> "sftp" then
```

Comment:
Why not just assume that the protocol is supported at this level, and let the implementation (liburl or tsNet) decide whether it likes it or not?